public/documentation/windows/build_crosswalk_for_windows_zh.md

```diff
@@ -2,7 +2,7 @@
 
 1. 下载并安装[Microsoft Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)。如果你在编译21版本之前的Crosswalk，则需要使用Microsoft Visual Studio 2013。
 
-2. 从http://git-scm.com/download/win处安装Windows平台的Git，并且确保将其添加到你的PATH中。设置你的用户环境变量。在Windows的开始菜单中，检索“环境变量”。或者，点击控制面板上的系统图标；然后进入高级系统设置中，点击环境变量按钮。你应该可以看到下列对话框：
+2. 从[win-git](http://git-scm.com/download/win)处安装Windows平台的Git，并且确保将其添加到你的PATH中。设置你的用户环境变量。在Windows的开始菜单中，检索“环境变量”。或者，点击控制面板上的系统图标；然后进入高级系统设置中，点击环境变量按钮。你应该可以看到下列对话框：
```

Comment:
Maybe, the commit message is not clear. I mean this one will generate a error URL address